### PR TITLE
91 UI implement bread crumb and header automation

### PR DESCRIPTION
**Pull request by mnyamunda-scottlogic**
This test mocks an api to only show 3 cities 
image

We click each link and assert that we can see the breadcrumb text.

### CODE REVIEW

File: air-quality-ui/e2e/summary.spec.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  await page.route("*/**/air-pollutant/forecast*", async (route) => {
+    await route.fulfill({ json: apiForecast });
+  });
+  await page.route(
+    "*/**/air-pollutant/measurements/summary*",
+    async (route) => {
+      await route.fulfill({ json: apiSummary });
+    }
+  );
+  await page.goto("http://localhost:5173/city/summary");
```
Comment (rstrange-scottlogic) [inline air-quality-ui/e2e/summary.spec.ts:14]: Can we pull this out into a environment variable or config item or something? We'll need to create a few playwright tests I imagine so having it in one easy to change place would be good.

File: air-quality-ui/e2e/summary.spec.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import { expect, test } from "@playwright/test";
```
Comment (rstrange-scottlogic) [inline air-quality-ui/e2e/summary.spec.ts:1]: Could we change the name to breadcrumbs.spec.ts or something? I think this is specifically for the breadcrumbs as opposed to the summary page?

Comment (mnyamunda-scottlogic) [reply air-quality-ui/e2e/summary.spec.ts:1]: It shall be done!

File: air-quality-ui/e2e/summary.spec.ts (hunk 1 above)
Comment (mnyamunda-scottlogic) [reply air-quality-ui/e2e/summary.spec.ts:14]: Makes sense, I can make this a baseURL inside playwright.config file for less repetition of code 😎

**Review by rstrange-scottlogic — DISMISSED** [review]

**Review by rstrange-scottlogic — APPROVED** [review]

**Pull request merged**